bloom-housing/bloom
# fix: storybook svgs don't appear

# Pull Request Template

## Issue

Related to but separate from #820 

## Description

Icons (SVGs) were not working on Storybook, locally or at our published Storybook URL, outside of the Icon story.
After lots of research, the way we were doing SVGs (basically as "external SVGs", symbol tags squished into one svg file and then indexed by ID) is deprecated and not supported by some tools (like Storybook). The preferred way of doing SVGs in a React application seems to be by component-tizing them, so that's what I've done here. This comes with multiple benefits:
1) Our SVGs can now take props and are thus more dynamic/extensible
2) We get Typescript auto-complete on icon types
3) Works in Storybook!
4) Not deprecated :)

## Type of change


## How Can This Be Tested/Reviewed?

I would mainly check stories w Icons like the Referral Application, you can also check out any icons in the application.

## Checklist:
